libimage/manifests/manifests_test.go

```diff
@@ -333,4 +333,8 @@ func TestPush(t *testing.T) {
 	options.Instances = append(options.Instances, otherListDigest)
 	_, _, err = list.Push(ctx, destRef, options)
 	assert.Nilf(t, err, "list.Push(four specified)")
+
+	options.ReplicateWithCompression = []string{"zstd"}
+	_, _, err = list.Push(ctx, destRef, options)
+	assert.Nilf(t, err, "list.Push(with replication for zstd specified)")
```

Comment:
(Absolutely non-blocking: `assert.NoError` IIRC produces better error messages if the test fails. OTOH keeping the existing style is also valuable.)

Reply:
Done.